### ISSUE TITLE
mix release failed after some upgrade 

### ISSUE THREAD

**Issue by zztczcx** [bitwalker/exrm#118]
I  release 3 times in the project and test in the rel  directory. (I used the exrm-test project for test)

for example  I can successfully release 0.0.1, 0.0.2, 0.0.3 . 
for test, I copy the  rel/test-0.0.3.tar.gz to release/0.0.3/test.tar.gz
and run bin/test upgrade "0.0.3".  it all worked well

but when I upgrade to 0.0.4 
I get this error: 

``` elixir
==> Generated .appup for test 0.0.3 -> 0.0.4
===> Missing beam file elf_format <<"/usr/lib/erlang/lib/hipe-3.11.2/ebin/elf_format.beam">>
Upfrom release version (0.0.3) for relup not found
==> Failed to build release. Please fix any errors and try again.
```

**Comment by zztczcx** [comment]
it seems like that the relup file are not generated in the release/0.0.4/

**Comment by bitwalker** [comment]
Fixed in master, soon to be on hex as 0.19.0
